cvmfs/file_processing/processor.cc

```diff
@@ -75,55 +75,39 @@ tbb::task* ChunkProcessingTask::execute() {
 void ChunkProcessingTask::Crunch(const unsigned char  *data,
                                  const size_t          bytes,
                                  const bool            finalize) {
```

Comment:
This is perhaps the hottest method regarding the parallel file processing of the backend. I believe it's worth measuring how much we hurt performance with several `memcpy()` calls here and in `Compressor::Deflate()` as well as the virtual function call to `::Deflate()`.